app/request.go

```diff
@@ -30,22 +55,22 @@ func GetRequestClient(ctx *cli.Context) (*RequestClient, error) {
 	return NewRequestClient(ct, conn), nil
 }
 
-func (c *RequestClient) getRequestStatus(requestID string) error {
+func (c *RequestClient) getRequestStatus(ctx *cli.Context, requestID string) (*request.RequestStatus, error) {
 	res, err := c.client.GetRequestStatus(c.ctx, &requestservice.GetRequestStatusRequest{
 		RequestId: requestID,
 	})
 	if err != nil {
-		return err
+		return nil, err
 	}
-	return PrintProto(res)
+	return res.RequestStatus, nil
 }
 
 func NewRequestCommand(getRequestClientFn GetRequestClientFn) (CommandOut, error) {
 
 	var c *RequestClient
 	return CommandOut{Command: &cli.Command{
 		Name:    "request",
-		Usage:   "Manage asynchronous requests",
+		Usage:   "Manage requests",
```

Comment:
I wonder if `long-running requests` could more clear for the user.

Reply:
`long-running` seems technically wrong, because some write operations we do will be very quick.

Reply:
Async operation makes sense as it is inline with the new APIs